security/certs.go

```diff
@@ -129,7 +142,7 @@ func RunCreateCACert(certsDir string, keySize int) error {
 // RunCreateNodeCert is the entry-point from the command-line interface
 // to generate node certs and keys:
 // - node.server.{crt,key}: server cert with list of dns/ip addresses
-// - node.{crt,key}: client cert with "node" as the Common Name.
+// - node.client.{crt,key}: client cert with "node" as the Common Name.
```

Comment:
if each node has `node` as their common name, will that mean that when logging an inbound connection from another node, all we can tell is that their name is "node"? Looks like it could be a good idea to have something unique here, though I'm not sure what that would be.